packages/@react-spectrum/combobox/intl/en-US.json

```diff
@@ -2,5 +2,7 @@
   "loading": "Loading...",
   "noResults": "No results",
   "clear": "Clear",
-  "invalid": "(invalid)"
+  "invalid": "(invalid)",
+  "valid": "valid"
```

Comment:
not quite sure why the "invalid" here has parens, those parens actually get announced in Talkback which sounds kinda weird. IMO we can remove them but if anyone remembers why we added them lemme know

Reply:
I think it was copied from v2 wayyyyy back when, can probably remove them

Reply:
gotcha, in that case I'll keep it then